server/src/types.ts

```diff
@@ -311,7 +311,12 @@ export type ContentDescription = {
   contentId: Uuid;
   name: string;
   type: ContentType;
-  parent: { contentId: string; type: ContentType } | null;
+  parent: {
+    contentId: Uuid;
+    type: ContentType;
+    name?: string;
+  } | null;
+  grandparentId?: Uuid | null;
```

Comment:
When we have type with an optional value that could be `null`, is to goal to use this convention?
- `undefined`: the value was not requested
- `null`: the value was requested and it does not exist